Dockerfile

```diff
@@ -27,15 +27,23 @@ RUN wget https://raw.githubusercontent.com/platformio/platformio-core-installer/
 	source ~/.platformio/penv/bin/activate && \
 	./bin/build-native.sh
 
-FROM frolvlad/alpine-glibc:glibc-2.31
+RUN cp /tmp/firmware/release/meshtasticd_linux_$(uname -m) /tmp/firmware/release/meshtasticd
 
-RUN apk --update add --no-cache g++ shadow && \
-	groupadd -g 1000 mesh && useradd -ml -u 1000 -g 1000 mesh
+##### PRODUCTION BUILD #############
 
-COPY --from=builder /tmp/firmware/release/meshtasticd_linux_x86_64 /home/mesh/
+FROM debian:bookworm-slim
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt-get update && \
+    apt-get -y install libc-bin libc6 libgpiod2 libyaml-cpp0.7
```

Comment:
We might want to add libulfius-dev liborcania-dev and libssl-dev to enable the web interface. Not sure if that belongs in this PR, though.